src/types.ts

```diff
@@ -47,44 +47,43 @@ export interface InvestScript {
 // tokens and reward tokens should be sent to
 export interface DivestScript {
   readonly address: string;
-  readonly signature:
-    | 'function divest(address recipient, uint256[] calldata receiptAmounts, uint256[] calldata rewardAmounts) external payable'
-    | 'function divest(address recipient, uint256[] calldata receiptAmounts, uint256[] calldata rewardAmounts, bytes calldata data) external payable';
+  readonly signature: string;
 }
 
 export interface StrategyLeg {
-  action: 'Borrow' | 'Swap to' | 'Deposit for' | 'Deposit to' | 'Stake on';
-  token: string; // address of the token, to be looked up in the `tokens` mapping
-  platformId: number; // Platform ID of strategy target
-  logoURI?: string;
+  readonly action: 'Borrow' | 'Swap to' | 'Deposit for' | 'Deposit to' | 'Stake on';
```

Comment:
Nit, but sort enum options alphabetically like we do the with each type's key names?